conf/messages.cy

```diff
@@ -14,9 +14,9 @@ getpagehelp.linkText = Help gyda’r dudalen hon (yn agor tab newydd)
 
 site.continue = Yn eich blaen
 
-label.day=Day
-label.month=Month
-label.year=Year
+label.day=Dydd
+label.month=Mis
+label.year=Blwyddyn
```

Comment:
I had "Blwydd". We will need to check later.

Reply:
I've updated this branch to the translation you've used to remove conflict